src/layouts/indexPageModes.js

```diff
@@ -0,0 +1,50 @@
+import Header from '../components/header/Header';
+import Container from '../components/container/Container';
+import Description from '../components/description/Description';
+
+export class IndexPageModes {
+  render() {
+    const container = document.createElement('section');
+
+    container.innerHTML = '';
+    container.classList.add('index-page__container');
+
+    container.style.gridArea = 'mode-description';
+    container.style.margin = '0 3rem';
+    container.style.alignSelf = 'start';
```

Comment:
I wonder if these styles could be kept in a separate file, just like in the case of the components?